terraform/projects/app-ecs-services/main.tf

```diff
@@ -109,5 +110,44 @@ resource "aws_s3_bucket" "config_bucket" {
   }
 }
 
+resource "aws_s3_bucket_policy" "config_bucket_policy" {
```

Comment:
Terraform thinks this object is changing everytime we run make apply which it shouldn't so we should fix this.

Reply:
We fixed this by adding the bucket policy version.